css_stylesheet.css

```diff
@@ -56,7 +55,8 @@ header {
 	font-family: "Courier New", serif;
 	color: #303030;
 	float: right;
-	margin-top: 55px;
+	margin-top: 2em;
+	font-size: 1.9em;;
 }
 
 .paragraphs {
```

Comment:
This is a flexbox with no additional flexbox behaviour defined. Is there a reason you need to use display:flex here as opposed to display:inline-block?

Reply:
Oh I see, you've applied flexbox rules later on to elements with additional classes. That's fine, but you might want to clear up that that's what you've done with a comment.